dali/base/dautils.cpp

```diff
@@ -466,21 +449,46 @@ void CDfsLogicalFileName::normalizeName(const char *name, StringAttr &res, bool
     }
     str.append("::");
     tailpos = str.length();
-    if (strstr(s,"::")!=NULL)
+    if (strstr(s,"::")!=nullptr)
         ERRLOG("Tail contains '::'!");
     normalizeScope(name, s, strlen(name)-(s-name), str, strict);
     str.toLowerCase();
     res.set(str);
 }
 
+bool isExternalFile(const char *name)
+{
+    // TODO Should check the name is a valid OS filename
+    if ('~' == *name) // allowed 1 leading ~
+        name++;
+
+    bool retVal = memicmp(name,EXTERNAL_SCOPE "::",sizeof(EXTERNAL_SCOPE "::")-1)==0;
+    const char *s=strstr(name,"::");
+    if (retVal && s)
+    {
+        const char *s1 = s+2;
+        const char *ns1 = strstr(s1,"::");
+        if (!ns1)
+            retVal = false;
+        else
+        {
+            StringBuffer nodename;
+            nodename.append(ns1-s1,s1);
```

Comment:
The old code (in normalizeName) used to clip here if !strict (which is how set called it)
Should keep the same in case break some files, that had a trailing space after IP.